code/go/0chain.net/blobbercore/blobbergrpc/blobber_grpc.pb.go

```diff
@@ -0,0 +1,101 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
+
```

Comment:
It is normally considered bad to keep generated files under version control. So do you really need these these generated files? 

Maybe you could add a script to generate them, and a section in the readme.md to explain how to generate them?

Reply:
Mostly for developer-experience, just like alot of other golang projects i have seen.

Generating at compile time will prevent any of the generated files from being out of date. I suggest we add github CI check to check if generated files are out of date and also regenerate the files while the service is being created in docker. This way any cons of keeping generated files in the codebase are eliminated, if there are other cons - please let me know.

Generated files could also fail in languages like c++ since the generated code will be OS specific. But since we are using golang this shouldnt be a problem since it has a runtime.

If you disagree with this i can remove the generated files as requested.

Reply:
I think it fine if, as you suggested, we can ensure that the generated files are kept up to date. 

Maybe add a .github/workfolw so that before merging the generated files are updated and confirmed to work.

Reply:
Latest changes should fix - https://github.com/0chain/blobber/issues/64